src/KM_Controls.pas

```diff
@@ -982,7 +989,7 @@   TKMDropCommon = class(TKMControl)
     property DropUp: Boolean read fDropUp write fDropUp;
     property ItemIndex: SmallInt read GetItemIndex write SetItemIndex;
 
-    property OnShow: TNotifyEvent read fOnShow write fOnShow;

```

Comment:
Renamed for better understanding (its actually works on show list, not on show Drop control)